protostar/commands/test/cheatcodes/skip_cheatcode.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Callable
+
+from protostar.starknet.cheatcode import Cheatcode
+
+from protostar.commands.test.test_environment_exceptions import (
+    SimpleSkippingReportedException,
+)
+
+
+class SkipCheatcode(Cheatcode):
+    @property
+    def name(self) -> str:
+        return "skip"
+
+    def build(self) -> Callable[..., Any]:
+        return self.skip
+
+    def skip(self, condition: bool = True, reason: str = "") -> None:
```

Comment:
```suggestion
    def skip(self, condition: bool = True, reason: Optional[str] = None) -> None:
```

wouldn't this be more semantically correct?